<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane

{

border-width: 2px;

border-color:white;

border-style:outset;

border-radius: 8px;

background-color: rgb(255, 255, 255,150);

font: 12pt &quot;楷体&quot;;	
}​
</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_linkControl">
       <attribute name="title">
        <string>客户端</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>客户端</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,1">
            <item>
             <widget class="QLineEdit" name="lineEdit_ca">
              <property name="placeholderText">
               <string>请输入服务器地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_cp">
              <property name="placeholderText">
               <string>请输入端口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="connectToSever">
              <property name="text">
               <string>连接服务端</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTextEdit" name="textEdit_cs">
              <property name="placeholderText">
               <string>请输入待发送数据</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="c_send">
              <property name="text">
               <string>发送目标数据</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTextEdit" name="textEdit_ca">
              <property name="placeholderText">
               <string>接受数据如下</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>输出：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_console_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_linkConfig">
       <attribute name="title">
        <string>服务端</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>网络设置</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,2,0,1">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>用户:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ca_2">
            <property name="placeholderText">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_cp_2">
            <property name="placeholderText">
             <string>密码</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="QTextEdit" name="textEdit_sa">
          <property name="placeholderText">
           <string>接受数据如下</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0,0,0">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>地址:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_sa">
            <property name="placeholderText">
             <string>服务器地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>端口:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_sp">
            <property name="placeholderText">
             <string>端口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>通信测试</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QTextEdit" name="textEdit_ss">
            <property name="placeholderText">
             <string>请输入待发送数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="s_send">
            <property name="text">
             <string>发送目标数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>选择客户端</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>输出：</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QTextEdit" name="textEdit_console"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="connectToClient">
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0450249, y1:0.181, x2:0.04, y2:0.75, stop:0 rgba(226, 231, 255, 255), stop:0.482587 rgba(162, 164, 255, 255), stop:0.965174 rgba(169, 175, 255, 255));
border-radius: 4px;
</string>
          </property>
          <property name="text">
           <string>监听端口</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
